threads-api/src/threads-api.ts

```diff
@@ -236,4 +245,26 @@ export class ThreadsAPI {
     const thread = res.data.data.data;
     return thread;
   };
+
+  getThreadLikers = async (postID: string) => {
+    if (this.verbose) {
+      console.debug('[fbLSDToken] USING', this.fbLSDToken);
+    }
+    const res = await axios.post<GetThreadLikersResponse>(
+      'https://www.threads.net/api/graphql',
+      new URLSearchParams({
+        lsd: this.fbLSDToken,
+        variables: `{"mediaID":"${postID}"}`,
+        doc_id: '9360915773983802',
+      }),
+      {
+        headers: {
+          ...this._getDefaultHeaders(),
+          'x-fb-friendly-name': 'BarcelonaPostPageQuery',
```

Comment:
```suggestion
```
nit, but I think the query name is wrong. Please remove this if you are not familiar with it!